patches/minecraft/net/minecraft/world/entity/LivingEntity.java.patch

```diff
@@ -549,6 +558,18 @@
     }
  
     public ItemStack m_5584_(Level p_21067_, ItemStack p_21068_) {
+@@ -3158,9 +_,8 @@
+    }
+ 
+    private void m_21063_(ItemStack p_21064_, Level p_21065_, LivingEntity p_21066_) {
+-      Item item = p_21064_.m_41720_();
+-      if (item.m_41472_()) {
```

Comment:
Don't change these two lines as that just makes it a larger patch, only change the call to `getFoodProperties`